claim/validations.py

```diff
@@ -226,13 +228,13 @@ def validate_claimitem_in_price_list(claim, claimitem):
 
 def validate_claimservice_in_price_list(claim, claimservice):
     errors = []
-    pricelist_detail = ServicesPricelistDetail.objects \
+    claim_date = claim.date_to if claim.date_to else claim.date_from
```

Comment:
Would have to check with business - I think @purusap mentioned some other date that is defined in this conditional.